### ISSUE TITLE
New Icon file-messages

### ISSUE THREAD

**Issue by lacrioque** [Templarian/MaterialDesign#7199]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

We needed an Icon to represent a list of sent messages for Administrators where all messages would be listed and should be exportable.
While trying out different styles and als combined icons we agreed upon contributing our design for an icon containing the already exisiting ```forum``` and the exisiting ```file```. 
Inspiratoin was the ```file-phone```

### Share a preview of your contribution.

![file-messages-preview](https://github.com/Templarian/MaterialDesign/assets/8501663/bb547509-a2cd-483e-92ba-d86633ed2626)

### Upload the icon source.

[file-messages.zip](https://github.com/Templarian/MaterialDesign/files/12284430/file-messages.zip)

**Comment by mririgoyen** [comment]
I don't think the double message bubbles work in this icon at native size. We'd probably want to limit it to a single bubble.

**Comment by Templarian** [comment]
![image](https://github.com/Templarian/MaterialDesign/assets/338885/407b089e-8663-4b1c-aaf2-1c053165c143)

Hmm, not sure how I feel about a simple bubble (or at least the above design).

**Comment by Simran-B** [comment]
Looks a bit stretched. I think the message modifier from e.g. the `speaker-message` icon works better:

![image](https://github.com/Templarian/MaterialDesign/assets/7819991/ca1b39de-f999-497d-8361-de54b39b944f)

**Comment by MrGrigri** [comment]
I agree @Simran-B however, we'd need to add the outlined variation as well and that won't work.

![file-message](https://github.com/Templarian/MaterialDesign/assets/4528551/40ceac66-e0b5-470d-a045-cb4fe9f239d1) ![file-message-outline](https://github.com/Templarian/MaterialDesign/assets/4528551/10eff208-80c9-425c-8bd1-ab9062137783)

![message-file](https://github.com/Templarian/MaterialDesign/assets/4528551/5d94690d-ef67-448b-b8da-3b9ef5281ef2) ![message-file-outline](https://github.com/Templarian/MaterialDesign/assets/4528551/2716f39b-362d-4579-979a-fb99c84088d1)

---


![file-message-outline](https://github.com/Templarian/MaterialDesign/assets/4528551/df8fcf70-578b-4d2f-b96a-5c570b4fcf35) ![file-message](https://github.com/Templarian/MaterialDesign/assets/4528551/4b062085-dc7f-420f-a0fc-ca7f9c21759c) ![message-file-outline](https://github.com/Templarian/MaterialDesign/assets/4528551/89833076-d1c5-4646-be6f-4f607b8ae605) ![message-file](https://github.com/Templarian/MaterialDesign/assets/4528551/0b6c5544-9f26-48ed-9ae4-c1cb1544e3c2)
